openedx/core/djangoapps/call_stack_manager/tests.py

```diff
@@ -108,6 +108,12 @@ def donottrack_func_child():
     # Should not be tracked
     ModelMixin.objects.all()
 
+@donottrack()
+def donottrack_check_with_return():
+    """ function that returns something i.e. a wrapped function returning some value
+    """
+    ModelMixin.objects.all()
```

Comment:
Nit: since you aren't asserting anything about the traced calls, you don't actually need this.